apps/docs/get-started/smtp.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: SMTP support
+description: "A guide to integrate Unsend with SMTP"
+icon: envelope  
+---
+
+## Prerequisites 
+
+You will need an API key and a verified domain to get the most out of this guide:
+
+- [API Key](https://app.unsend.dev/dev-settings/api-keys)
+- [Verified Domain](https://app.unsend.dev/domains)
+
+## SMTP credentials
+
+To set up your SMTP integration, you'll need to provide the following credentials:
+
+- **Host:** ```smtp.unsend.dev```
+- **Port:** ```465```, ```587```, ```2465```, or ```2587```
+- **Username:** ```unsend```
+- **Password:** ```YOUR-API-KEY```
```

Comment:
can you add some sample code here? like using nodemailer. other than that looks good